codex-cli/tests/terminal-chat-input-multiline.test.tsx

```diff
@@ -40,6 +40,8 @@ describe("TerminalChatInput multiline functionality", () => {
       openModelOverlay: () => {},
       openApprovalOverlay: () => {},
       openHelpOverlay: () => {},
+      // toggle flex-mode mid-session
```

Comment:
Should remove useless comments

Reply:
Yeah, those were excessive and uninformative. Removed a bunch of them, thx.